components/tidb_query_datatype/src/codec/mysql/time/mod.rs

```diff
@@ -2735,6 +2735,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::excessive_precision)]
```

Comment:
I don't get it, what can trigger it?

Reply:
Some test cases use float value which excess precision, not sure if we need to modify these tests.

Reply:
/cc @iosmanthus PTAL

Reply:
LGTM